app/view_models/workarea/storefront/product_templates/package_view_model.rb

```diff
@@ -6,27 +6,15 @@ class ProductTemplates::PackageViewModel < ProductViewModel
                :msrp?, :msrp, :msrp_min, :msrp_max, :sale_starts_at,
                :sale_ends_at, to: :pricing
 
-      def primary_image
-        product_image = super
-        return product_image unless product_image.placeholder?
-
-        @primary_image ||=
-          packaged_products
-          .map(&:images)
-          .flatten
-          .reject(&:placeholder?)
-          .first ||
-          Catalog::ProductPlaceholderImage.cached
-      end
-
+      # this isn't used, no idea why it's here.
```

Comment:
This is going to be a major release anyways, let's just get rid of it. Prob a relic from a diff UI long past.